test-case/manual/wov-script/wov-test.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+rm -rf blob
```

Comment:
This is the only command here, all the rest is just initializing variables so it's out of place. Move it before the first command, I think this means before the `_save_def` invocation